build.sbt

```diff
@@ -1000,7 +1001,13 @@ lazy val ui = (project in file("ui/server"))
   .settings(
     name := "nussknacker-ui",
     buildUi :=  {
-      runNpm("run build", "Client build failed", (compile / crossTarget).value)
+      if (useFEBuildTarget) {
+        val feDistDirectory = file("ui/client/dist")
+        val feDistFiles: Seq[File] = (feDistDirectory ** "*").get()
+        IO.copy(feDistFiles pair Path.rebase(feDistDirectory, (compile / crossTarget).value / "classes" / "web" / "static"), CopyOptions.apply(overwrite = true, preserveLastModified = true, preserveExecutable = false))
```

Comment:
Maybe we can just have this code regardless of useFEBuildTarget and remove OUTPUT_PATH settings to be consistent?